lib/e3/ecl/fortio.h

```diff
@@ -0,0 +1,183 @@
+#ifndef ECL_FORTIO_H
+#define ECL_FORTIO_H
+
+#include <stdio.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/*
```

Comment:
It does not matter for some time - but I think it should be an open question whether the `fortio.h` should be publically exposed at all. I am leaning to no.